### ISSUE TITLE
Auto-updater error

### ISSUE THREAD

**Issue by ElTyranos** [ross-g/io_pdx_mesh#98]
**Describe the bug**
```
AttributeError: 'str' object has no attribute 'get'
Traceback (most recent call last):
  File "C:\Users\Pierre\AppData\Roaming\Blender Foundation\Blender\4.2\extensions\user_default\io_pdx_mesh\pdx_blender\blender_ui.py", line 755, in draw
    github.LATEST_URL.get("blender", github.LATEST_RELEASE)
    ^^^^^^^^^^^^^^^^^^^^^
```

**To Reproduce**
1. Press M to merge vertices
2. Merge by distance
3. The console issue appears

**Software (please complete the following information):**
 - OS: Windows 11
 - Blender 4.2.1

**Comment by ross-g** [comment]
Could you send me your user settings file from this folder? `%localappdata%\io_pdx_mesh`
Does deleting the settings file (or specifically, deleting the `"github_latest_url":` data) solve the problem?

**Comment by ElTyranos** [comment]
Deleting fixed it. I forgot to back it up tho :-|

**Comment by ross-g** [comment]
Thanks for confirming 👍 don't worry about forgetting to back up, there's nothing really important in there.
Just some cached data and your last export/import files.
eg
```
{
    "app": "C:\\Program Files\\Blender Foundation\\Blender 4.2\\4.2\\python\\bin\\python.exe",
    "config_dir": "C:\\Users\\Ross\\AppData\\Local\\io_pdx_mesh",
    "config_path": "C:\\Users\\Ross\\AppData\\Local\\io_pdx_mesh\\settings.json",
    "github_latest_notes": "2024-09-23\r\nRelease version: 0.91\r\n### Features:\r\n- Blender 4.2 support\r\n- Blender 4.2 \"extensions\" support\r\n- Added Vic3 materials presets\r\n- Updater support for split download packages\r\n\r\n### Bugfixes:\r\n- Shader nodes update to support Blender 4.0 (see #86)\r\n- Custom normals update to support Blender 4.1 (see #92)\r\n\r\n**Full Changelog**: https://github.com/ross-g/io_pdx_mesh/compare/0.9...0.91\r\n\r\nThanks to @Tetrino for putting up some fixes for download to support Blender 4.0 & 4.1 while I've been delayed! Much appreciated. \u2b50\r\n",
    "github_latest_url": {
        "0.91_io_pdx_mesh": "https://github.com/ross-g/io_pdx_mesh/releases/download/0.91/0.91_io_pdx_mesh-latest_release.html",
        "blender": "https://github.com/ross-g/io_pdx_mesh/releases/download/0.91/blender-io_pdx_mesh.zip",
        "maya": "https://github.com/ross-g/io_pdx_mesh/releases/download/0.91/maya-io_pdx_mesh.zip"
    },
    "github_latest_version": 0.91,
    "last_export_anim": "C:\\Users\\Ross\\Desktop\\pig_idleA.anim",
    "last_export_mesh": "D:\\SteamLibrary\\steamapps\\common\\Victoria 3\\game\\gfx\\models\\buildings\\wonders\\giza_necropolis_01\\wonder_giza_necropolis_01.mesh",
    "last_import_anim": "C:\\Program Files (x86)\\Steam\\steamapps\\common\\Crusader Kings III\\game\\gfx\\models\\pets\\horses\\horse_01\\ep2_horse_walk.anim",
    "last_import_mesh": "C:\\Program Files (x86)\\Steam\\steamapps\\common\\Crusader Kings III\\game\\gfx\\models\\pets\\horses\\horse_01\\ep2_horse_01.mesh",
    "last_set_engine": "crusader_kings_3",
    "last_update_check": "2024-09-29"
}
```

I think the bug you found could happen if you updated to the new version, switched back to the old version temporarily and then used the new version again. I just need to bear that in mind when I change settings data in future.
